### PR TITLE
MongoDB: Remove `OrganizationalUnit` and default `Organization`

### PR DESCRIPTION
**Pull request by faem**
Signed-off-by: Mohammad Fahim Abrar <fahimabrar@appscode.com>

### CODE REVIEW

File: apis/kubedb/v1alpha2/mongodb_helpers.go [hunk 1]
```diff
@@ -438,44 +438,64 @@ func (m *MongoDB) SetTLSDefaults() {
...
-			Organizations:       []string{KubeDBOrganization},
-			OrganizationalUnits: []string{string(MongoDBClientCert)},
+			Organizations: defaultClientOrg,
 		},
 	})
+
+	defaultExporterOrg := []string{KubeDBOrganization}
+	_, cert = kmapi.GetCertificate(m.Spec.TLS.Certificates, string(MongoDBMetricsExporterCert))
+	if cert != nil && cert.Subject != nil && cert.Subject.Organizations != nil {
+		defaultExporterOrg = []string{}
```
Comment (tamalsaha) [inline apis/kubedb/v1alpha2/mongodb_helpers.go:488]: Why not just assign `cert.Subject.Organizations` ?

Comment (faem) [reply apis/kubedb/v1alpha2/mongodb_helpers.go:488]: It is already assigned. So passing an empty string. But setting `cert.Subject.Organizations` will also work as we use `stringSetMerger{}` to merge later.

Comment (tamalsaha) [reply apis/kubedb/v1alpha2/mongodb_helpers.go:488]: Let's assign cert.Subject.Organizations . That is less confusing.

Comment (faem) [reply apis/kubedb/v1alpha2/mongodb_helpers.go]: updated

**Review by tamalsaha — APPROVED** [review]

**Pull request merged**